lib/crowbar/client/command/node/rename.rb

```diff
@@ -35,6 +35,8 @@ def execute
                 say "Successfully updated alias for #{args.name}"
               when 404
                 err "Node does not exist"
+              when 500
+                err "Rename for #{args.name} failed. Node alias not unique? "
```

Comment:
I'm not convinced that the rails API should return 500 for this. Sounds like a crash on the rails side...